var/spack/repos/builtin/packages/sys-sage/package.py

```diff
@@ -3,23 +3,79 @@
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
-
 from spack.package import *
 
 
 class SysSage(CMakePackage):
-    """A library for capturing hadrware topology and attributes of compute systems."""
+    """A library for capturing hardware topology and attributes of compute systems."""
 
-    homepage = "https://github.com/stepanvanecek/sys-sage"
-    url = "https://github.com/stepanvanecek/sys-sage/archive/refs/tags/v0.1.1-alpha.2.tar.gz"
-    git = "https://github.com/stepanvanecek/sys-sage.git"
+    homepage = "https://github.com/caps-tum/sys-sage"
+    url = "https://github.com/caps-tum/sys-sage/archive/refs/tags/v0.4.3.tar.gz"
+    git = "https://github.com/caps-tum/sys-sage.git"
 
-    maintainers("stepanvanecek")
+    maintainers = ["stepanvanecek"]
```

Comment:
Please revert this change, `maintainers` has been a directive for a while:
```suggestion
    maintainers("stepanvanecek")
```